tests/e2e/api/routing.js

```diff
@@ -661,7 +661,7 @@ describe('routing', () => {
         ),
       ]).then(results => {
         expect(results[0].includes('Found. Redirecting to')).toBe(true);
-        expect(results[1].includes('DOCTYPE html')).toBe(true);
+        expect(results[1].includes('Found. Redirecting to')).toBe(true);
```

Comment:
I think we should change the `path` in the test rather than the expected outcome. Maybe just update line 656 to `path: '/'` and see if that works with the previous expectation?